### PR TITLE
Support IERC1155

### PR DESCRIPTION
**Pull request by 0x0Louis**

### CODE REVIEW

File: src/LBToken.sol [hunk 1]
```diff
@@ -120,16 +119,18 @@ contract LBToken is ILBToken {
         address _from,
         address _to,
         uint256 _id,
-        uint256 _amount
-    ) public virtual override checkAddresses(_from, _to) checkApproval(_from, msg.sender) checkLBTokenSupport(_to) {
-        address _spender = msg.sender;
-
+        uint256 _amount,
+        bytes calldata
```
Comment (Mathieu-Be) [inline src/LBToken.sol:123]: Missing the natspec

File: src/LBToken.sol [hunk 2]
```diff
@@ -138,31 +139,29 @@ contract LBToken is ILBToken {
         address _from,
         address _to,
         uint256[] calldata _ids,
-        uint256[] calldata _amounts
+        uint256[] calldata _amounts,
+        bytes calldata
```
Comment (Mathieu-Be) [inline src/LBToken.sol:143]: Missing the natspec

File: src/LBToken.sol (hunk 1 above)
Comment (Mathieu-Be) [reply src/LBToken.sol:123]: Nvm it doesn't accept unnamed variables

**Review by Mathieu-Be — APPROVED** [review]

File: src/LBToken.sol [hunk 3]
```diff
@@ -111,7 +100,10 @@ contract LBToken is ILBToken {
         _setApprovalForAll(msg.sender, _spender, _approved);
     }
 
-    /// @notice Transfers `_amount` token of type `_id` from `_from` to `_to`
+    /// @notice Transfers `_amount` token of type `_id` from `_from` to `_to`. If the `to` address is a contract, it will
```
Comment (pecio222) [inline src/LBToken.sol:103]: `If the `to` address is a contract, it will` is doubled

**Review by pecio222 — APPROVED** [review]

**Pull request merged**